Inputs/Input_Info.py

```diff
@@ -11,9 +11,9 @@
 INPUT_m4l_high = 160.0
 
 # Lumi for each year
-Lumi_2016 = 36.6
-Lumi_2017 = 41.5
-Lumi_2018 = 59.7
+Lumi_2016 = 36.33
```

Comment:
I still see .33 instead of .31 no? @ram1123

Reply:
0.33 is the latest number. Earlier it was 36.31 I updated this to 36.33

Below attaching the screenshot :

![Screenshot 2022-08-25 at 18 20 23](https://user-images.githubusercontent.com/5220316/186717943-31f52b24-f192-4ebf-b7e1-fb492817cc41.png)